include/openssl/base.h

```diff
@@ -202,7 +202,7 @@ extern "C" {
 // against multiple revisions of BoringSSL at the same time. It is not
 // recommended to do so for longer than is necessary.
 
-#define AWSLC_API_VERSION 16
+#define AWSLC_API_VERSION 17
```

Comment:
Nit:
I added some notes to tell the change from 16 - 17 in https://github.com/awslabs/aws-lc/pull/454/files#diff-1ee2dcc1eaa98a4d7ac8d2571dabcf492f6cadeeedf8edd7c2bd6faeca619920R205-R207
```
#define AWSLC_API_VERSION 16
// Bump version from 16 to 17.
//   - Import new APIs |EVP_aes_128/256_cbc_hmac_sha1/256| from OpenSSL.
#define AWSLC_API_VERSION 17
```